<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1040</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>能源设备仿真模拟软件</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="widget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>基准模拟</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="8,2">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>GroupBox</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="1">
                <widget class="QPushButton" name="pushButton_browse_idf">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>2</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>选择IDF文件</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLineEdit" name="lineEdit_weather">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>2</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="pushButton_browse_control">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>2</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>选择控制文件</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="pushButton_browse_weather">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>2</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>选择气象文件</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLineEdit" name="lineEdit_idf">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>2</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLineEdit" name="lineEdit_control">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>2</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,6,2">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_run_baseline">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>开始基准模拟</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="7,3">
            <item>
             <widget class="QTableWidget" name="tableWidget_baseline">
              <column>
               <property name="text">
                <string>Timestamp</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ChW_Pump_Flow</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>CndW_Pump_Flow</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ChW_In_Temp</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ChW_Out_Temp</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>CndW_In_Temp</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>CndW_Out_Temp</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>COP</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Total_Power</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="textBrowser_log"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_optimized">
       <attribute name="title">
        <string>优化模拟</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="3,7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="3,7">
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>种群大小</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_pop_size"/>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>交叉率</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="doubleSpinBox_crossover"/>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>最大迭代次数</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_max_gen"/>
                </item>
               </layout>
              </item>
              <item row="1" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>变异率</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="doubleSpinBox_mutation"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>变量1范围</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_var1_range"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>变量2范围</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_var2_range"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>变量3范围</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_var3_range"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>变量4范围</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_var4_range"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="7,3">
            <item>
             <widget class="QTableWidget" name="tableWidget_optimized">
              <column>
               <property name="text">
                <string>Timestamp</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ChW_Pump_Flow</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>CndW_Pump_Flow</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ChW_In_Temp</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ChW_Out_Temp</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>CndW_In_Temp</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>CndW_Out_Temp</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>COP</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Total_Power</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="textBrowser_opt"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,2,1,2,1">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_run_optimized">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>开始优化模拟</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar_ga">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>50</height>
             </size>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
